### PR TITLE
Fixed missing comment field issue.

### PR DESCRIPTION
**Pull request by ZenabKhan**
When running `make loaddata` I was getting an error of missing comment field in the event. 
this just the fix for that.

### CODE REVIEW

File: event/serializers.py [hunk 1]
```diff
@@ -45,14 +45,14 @@ class EventSerializer(serializers.ModelSerializer):
     organisation = OrganisationSerializer()
     location = EventLocationSerializer()
     sponser = EventSponserSerializer(many=True)
-    comments = EventCommentSerializer(many=True)
+    comment = EventCommentSerializer(many=True)
```
Comment (mashhoodr) [inline event/serializers.py]: @ZenabKhan this should be comments no?

Comment (ZenabKhan) [reply event/serializers.py]: @mashhoodr all other fields are with a signular name. 
So just made it consistent. Should I revert back?

**Comment by usmandap** [comment]
@mashhoodr This pr contains a new dump which contains only 1 event,organisation and a default superuser admin. We will update the dump later with more populated data currently we created a new fresh db dump in order to fix the load data issue.

File: event/serializers.py (hunk 1 above)
Comment (mashhoodr) [reply event/serializers.py]: Actually fix all of htem, if a variable has multiple (array) then it should be plural

Comment (ZenabKhan) [reply event/serializers.py]: Fixed.

**Pull request merged**